fw/http_parser.c

```diff
@@ -3295,15 +3295,26 @@ __req_parse_x_tempesta_cache(TfwHttpMsg *hm, unsigned char *data, size_t len)
 		/* "X-Tempesta-Cache" ":" method */
 		TRY_STR_fixup(&TFW_STR_STRING("get"), Req_I_X_Tempesta_Cache,
 			      I_Tempesta_Cache_get);
-		return CSTR_NEQ;
+		/* If the method is not "GET" just ignore the rest of the
+		 * header line. */
+		__FSM_I_JMP(I_Tempesta_Cache_skip);
 	}
 
 	__FSM_STATE(I_Tempesta_Cache_get) {
 		if (likely(IS_CRLF(c))) {
```

Comment:
I've changed the parsing logic so that it will also accept a comma or whitespace in addition to CRLF after the "get" string (and ignore the rest of the string as before).

Reply:
Why do we need to accept comas? The task requires to accept `get` only as the header value. Allowing spaces is good sine all HTTP headers allow spaces at the end and sometimes headers are generated automatically, so extra spaces may appear. Form the other hand, `X-Tempesta-Cache: get , \r\n` doesn't look correct. I believe we should not allow comas.